v2/api-validator/src/server/app.ts

```diff
@@ -83,8 +83,15 @@ export class WebApp {
   }
 }
 
-function ajvRegisterJsonSchemaStandardStringFormats(ajv: Ajv) {
+function ajvRegisterJsonSchemaStandardNumberFormats(ajv: Ajv) {
```

Comment:
Sorry, but these are not only number formats, and not only standard formats. Call this function `ajvRegisterJsonSchemaDataFormats`.